src/zenml/zen_server/deploy/helm/values.yaml

```diff
@@ -194,6 +194,8 @@ zenml:
     # sslCert: /path/to/client-cert.pem
     # sslKey: /path/to/client-key.pem
     # sslVerifyServerCert: True
+    backupStorageSize: 1Gi
+    # storageClass: standard
```

Comment:
`backupStorageClass` is a better name for this, otherwise it can be confused with the volume used for the sqlite DB (which we don't support yet, but maybe we will).

Reply:
Good point, I agree.